apps/api/src/domain/models/song/commands/translate-song-title/translate-song-title.command-handler.ts

```diff
@@ -22,6 +22,7 @@ import { TranslateSongTitle } from './translate-song-title.command';
 
 @CommandHandler(TranslateSongTitle)
 export class TranslateSongTitleCommandHandler extends BaseUpdateCommandHandler<Song> {
+    // TODO do we need this?
```

Comment:
Remove the constructor if it is not needed.

Reply:
done